contribs/gnodev/pkg/dev/node_state_test.go

```diff
@@ -0,0 +1,189 @@
+package dev
+
+import (
+	"context"
+	"strconv"
+	"testing"
+
+	emitter "github.com/gnolang/gno/contribs/gnodev/internal/mock"
+	"github.com/gnolang/gno/contribs/gnodev/pkg/events"
+	"github.com/gnolang/gno/gno.land/pkg/gnoclient"
+	"github.com/gnolang/gno/gno.land/pkg/gnoland"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNodeMovePreviousTX(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	const callInc = 5
+
+	node, emitter := testingCounterRealm(t, callInc)
+
+	t.Run("Prev TX", func(t *testing.T) {
+		err := node.MoveToPreviousTX(ctx)
+		require.NoError(t, err)
```

Comment:
I'm on a nitpicking role here
```suggestion
		require.NoError(t, node.MoveToPreviousTX(ctx))
```

Reply:
i know, but i prefere having consistency and always have 
-- execution
-- test